embrace_mobile/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "display_name": "Embrace Mobile",
+  "maintainer": "support@embrace.io",
+  "manifest_version": "1.0.0",
```

Comment:
This needs to be migrated to Manifest 2.0.  Please see the email you should have received that describes the process.